integrations/tensorflow/compiler/Passes.cpp

```diff
@@ -25,56 +25,64 @@
 
 namespace mlir {
 namespace iree_compiler {
+namespace TF {
+
+void registerAllDialects(mlir::DialectRegistry &registry) {
+  registry.insert<tf_tensorlist::TFTensorListDialect,
+                  tf_strings::TFStringsDialect>();
+}
 
 // All IREE-specific passes that lower TF representations before reaching the
 // IREE core should go here.
-void createIreeTfImportPipeline(OpPassManager &pm) {
```

Comment:
@stellaraccident this (and possibly other files) will conflict with https://github.com/google/iree/pull/4043

Reply:
FIFO - go ahead and land and I'll rebase and likely submit on the OSS side (thought there were going to be more internal entanglements than I thought so started there). I can wait ~a day.

Reply:
SGTM. I just resolved the merge conflicts with @rsuderman 's https://github.com/google/iree/pull/3941. Planning on merging once checks come back green.

Reply:
Thanks - I was going to offer to help land it, but it seems like you've already done the work :)

Reply:
> Sorry for the delayed reviews. This generally LGTM.
> 
> * Any thoughts, @rsuderman ?
> * This has some conflicts with tip of tree and with @stellaraccident 's pending #4043 . How do you all want to proceed?

Looks fine with me assuming it does not conflict too much with Stella's changes. I initially hesitated at removing the metadata until I realized its long after I need it for shape inference.

Reply:
It conflicts but I'll rebase (and had already locally fixed up some of the style issues so hopefully not too bad) and send a fresh PR on the main branch when ready.